File: trunk/quests/airplane/71006.pl
#Bzzazzt - #2
sub EVENT_DEATH_COMPLETE {
  my $x = $npc->GetX();
  my $y = $npc->GetY();
  my $z = $npc->GetZ();
  my $h = $npc->GetHeading();
  quest::spawn2(71106,0,0,$x+10,$y,$z,$h); #Bazzazzt #2-1
  quest::spawn2(71106,0,0,$x-10,$y,$z,$h); #Bazzazzt #2-2
  quest::spawn2(71106,0,0,$x   ,$y,$z,$h); #Bazzazzt #2-3
}

File: trunk/quests/airplane/71006.lua
function event_death_complete(e)
	eq.spawn2(71106,0,0,e.self:GetX() + 10,e.self:GetY(),e.self:GetZ(),e.self:GetHeading()); --Bazzazzt --2-1
	eq.spawn2(71106,0,0,e.self:GetX() - 10,e.self:GetY(),e.self:GetZ(),e.self:GetHeading()); --Bazzazzt --2-2
	eq.spawn2(71106,0,0,e.self:GetX(),e.self:GetY(),e.self:GetZ(),e.self:GetHeading()); --Bazzazzt --2-3
end

-------------------------------------------------------------------------------------------------
-- Converted to .lua using MATLAB converter written by Stryd
-- Find/replace data for .pl --> .lua conversions provided by Speedz, Stryd, Sorvani and Robregen
-------------------------------------------------------------------------------------------------
